src/com/yetanalytics/lrs_admin_ui/handlers.cljs

```diff
@@ -396,16 +399,19 @@
         :as db} :db} _]
    (let [{:keys [username] :as new-account}
          (::db/new-account db)]
-     {:http-xhrio {:method          :post
-                   :uri             (httpfn/serv-uri
-                                     server-host
-                                     "/admin/account/create")
-                   :response-format (ajax/json-response-format {:keywords? true})
-                   :params          new-account
-                   :format          (ajax/json-request-format)
-                   :on-success      [:accounts/create-success username]
-                   :on-failure      [:accounts/create-error]
-                   :interceptors    [httpfn/add-jwt-interceptor]}})))
+     (if (valid? ::input/valid-new-account new-account)
+       {:http-xhrio {:method          :post
+                     :uri             (httpfn/serv-uri
+                                       server-host
+                                       "/admin/account/create")
+                     :response-format (ajax/json-response-format {:keywords? true})
+                     :params          new-account
+                     :format          (ajax/json-request-format)
+                     :on-success      [:accounts/create-success username]
+                     :on-failure      [:accounts/create-error]
+                     :interceptors    [httpfn/add-jwt-interceptor]}}
+       {:fx         [[:dispatch [:notification/notify true
+                                 "Username or Password did not meet requirements."]]]}))))
```

Comment:
Why is "Password" capitalized here? It makes it seem like I'm reading German.

Reply:
it's fine, they're separate fields.